index.html

```diff
@@ -36,14 +36,42 @@
+
+<p>Edge Protected: X is edge-protected if every segment of X is strongly delaunay.</p>
+
+<p>Delaunay tetrahedralization: A Delaunay tetrahedralization D of V is a tetrahedralization such that all simplices of D are delaunay.</p>
+
+<p>Contrained Delaunay: A tetrahedron t formed by vertices of X is constrained delaunay if its circumsphere encloses no vertex of X which is visible from any point in the relative interior of t.</p>
+
+<p>Constrained tetrahedralization: A tetrahedralization T is a constrained tetrahedralization of X if T and X have exactly the same vertices, and the tetrahedra in T cover the convex hull of V . every segment of X is an edge of T , every facet of X is a union of triangular faces of T .</p>
+
+<p>CDT: A constrained tetrahedralization T of X is said to be a constrained delaunay tetrahedralization (CDT) of X if each tetrahedron of T is constrained delaunay.</p>
+
+<p>Local Degenracy: Let D be the delaunay tetrahedralization of the vertices of X, t a tetrahedron in D and t' an adjacent tetrahedron of t (sharing a face with t), V the set of vertices of t, t'. If all vertices of V lie on a common sphere, V is called a local degeneracy of X.</p>
+
+<h1>
+<a id="the-two-alternatives" class="anchor" href="#the-two-alternatives" aria-hidden="true"><span class="octicon octicon-link"></span></a>The Two alternatives</h1>
+
+<p>The paper highlights two ways to get to be sure that a CDT exists.The implementation is done using the second alternative because using second alternative the facet recovery(last step) can be carried out efficiently.</p>
+
+<p>First Way: If X is edge protected ,then X has a CDT.  </p>
+
+<p>Second Way: If D(delaunay tetrahedralization) contains no local degeneracy and contains all segments of X, then the CDT of X exists.</p>
+
 <h2>
 <a id="algorithm-overview" class="anchor" href="#algorithm-overview" aria-hidden="true"><span class="octicon octicon-link"></span></a>Algorithm overview</h2>
 
-<p>Algorithm is based on <a href="www.cs.berkeley.edu/%7Ejrs/papers/cdt.pdf">Shewchuck's theorem</a> which states that: 'If a PLC <em>X</em> is <em>edge protected</em> and there are no cospherical 5-points in vertex set of <em>X</em> then CDT of X exists'. This algorithm first transforms the input <em>Piecewise linear complex</em>(or PLC) to a PLC which is edge protected and then removes any degenerate point set and then tranformed PLC is guarnateed to have CDT. Precisely,<br>
+<p>Algorithm is based on <a href="www.cs.berkeley.edu/%7Ejrs/papers/cdt.pdf">Shewchuck's theorem</a> which states that: 'If a PLC <em>X</em> is <em>edge protected</em> and there are no cospherical 5-points in vertex set of <em>X</em> then CDT of X exists'.</p>
+
+<p>This algorithm first transforms the input <em>Piecewise linear complex</em>(or PLC) to a PLC which is edge protected and then removes any degenerate point set and then tranformed PLC is guarnateed to have CDT. Precisely,<br>
```

Comment:
@HarshitJn add a fact that input PLC and PLC obtained after transformation are _topologically equivalent_. BTW, _topological equivalence_ is a standard term so no need to define it.